webapp/pages/profile/_id/_slug.vue

```diff
@@ -331,6 +331,10 @@ export default {
     },
   },
   methods: {
+    removePostFromList(index) {
+      this.activePosts.splice(index, 1)
```

Comment:
From where you get the `index` parameter, dear? @roschaefer 
After you deleted it from the `for loop` above …

Reply:
The index parameter is still there! You pass the function `removePostFromList` and it has the signature of expecting one argument. When it's called it gets called with the required index param.

Reply:
![get called with the right arguments](https://i.imgflip.com/35pl5p.jpg)